webui/react/src/components/kit/UserBadge.tsx

```diff
@@ -1,32 +1,25 @@
 import React from 'react';
 
-import Avatar, { Props as AvatarProps } from 'shared/components/Avatar';
-import useUI from 'shared/contexts/stores/UI';
-import { getDisplayName, UserNameFields } from 'utils/user';
+import { UserNameFields } from 'utils/user';
 
-import css from './UserBadge.module.scss';
+import Nameplate from './Nameplate';
+import UserAvatar from './UserAvatar';
 
-export interface Props extends Omit<AvatarProps, 'darkLight' | 'displayName'> {
+export interface Props {
+  className?: string;
   compact?: boolean;
   user: UserNameFields;
 }
 
-const UserBadge: React.FC<Props> = ({ user, compact, ...rest }) => {
-  const { ui } = useUI();
-  const displayName = getDisplayName(user);
-
-  const avatar = <Avatar {...rest} darkLight={ui.darkLight} displayName={displayName} />;
-  const classnames = [css.avatarCard];
-  if (compact) classnames.push(css.compact);
-
+const UserBadge: React.FC<Props> = ({ user, compact, className }) => {
```

Comment:
We should remove UserBadge from the UI kit since it's just a thin wrapper around Nameplate.